topics/dev/tutorials/core-contributing/tutorial.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: tutorial_hands_on
+
+title: "Contributing to Galaxy Core"
```

Comment:
I wonder whether this should be more specific, like "Contributing a Feature to Galaxy Core"? The other core tutorial is "Debugging Galaxy" - which is also contributing (via fixing bugs and writing tests). But maybe it's fine as is.